LivingCity/traffic/b18CommandLineVersion.cpp

```diff
@@ -40,15 +40,15 @@ void B18CommandLineVersion::runB18Simulation() {
   const float endDemandH = settings.value("END_HR", 12).toFloat();
   const bool showBenchmarks = settings.value("SHOW_BENCHMARKS", false).toBool();
   const parameters simParameters {
-      settings.value("A",0.9423627817042464).toFloat(),
-      settings.value("B",1.4927856811009153).toFloat(),
-      settings.value("T",1.1675957997007158).toFloat(),
-      settings.value("s_0",3.637803967100891).toFloat()};
+      settings.value("A",0.557040909258405).toDouble(),
+      settings.value("B",2.9020578588167).toDouble(),
+      settings.value("T",0.5433027817144876).toDouble(),
+      settings.value("s_0",1.3807498735425845).toDouble()};
 
 
   std::cout << "b18CommandLineVersion received the parameters "
-            << "[a: " << simParameters.a 
```

Comment:
I would use `a` and `b` instead of `A` and `B` because that is the proper notation in the Intelligent Driver Model.